README.md

```diff
@@ -6,13 +6,16 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 > [!WARNING]
->  Fauna is decomissioning FQL v4 on June 30, 2025. See the [v4 end of life (EOL)
+>  Fauna is decomissioning FQL v4 on June 30, 2025.
+>
+> This driver is not compatible with FQL v10, the latest version. Fauna accounts
+> created after Month DD, YYYY must use FQL v10. Ensure you migrate existing
```

Comment:
I'm assuming we'll replace this with the correct date prior to merging?

Reply:
Yep — This is just a placeholder until we have a firm date. Thanks!